lib/vagrant-openshift/action/install_docker.rb

```diff
@@ -44,7 +44,12 @@ def call(env)
           )
 
           # Configure the Docker daemon
-          sudo(@env[:machine], "SSH_USER='#{ssh_user}' #{home}/configure_docker.sh", :timeout=>60*30)
+          if @options[:skip_volume_group]
```

Comment:
1) I don't see set -o nounset before your changes:

https://github.com/openshift/vagrant-openshift/blob/a2bf2d9c93922d3e30d74f966c544f454c9bc6b1/lib/vagrant-openshift/action/install_origin_base_dependencies.rb

2) Even if it was there, the intention was that it would work without a vg by default.

That's the goal I am trying to keep, by default it works with a standard image.